packages/gafl-webapp-service/src/pages/contact/name/__tests__/update-transaction.spec.js

```diff
@@ -0,0 +1,73 @@
+import { NAME } from '../../../../uri.js'
+import updateTransaction from '../update-transaction.js'
+
+const VALID_PAYLOAD = {
+  'first-name': 'Luke',
+  'last-name': 'Skywalker'
+}
```

Comment:
Don't use static values defined outside of a test in multiple tests. It's fine to have a function that gives you a value, but the problem with these const values is that they can change. Fresh test, fresh mocks.